business-registry-dashboard/app/composables/useBrdModals.ts

```diff
@@ -31,6 +31,16 @@ export const useBrdModals = () => {
     })
   }
 
+  function openMagicLinkModal (title: string, description: string) {
```

Comment:
New blueprint for the magic link dialogs. This is to avoid creating one for each error (one for link expired, one for already added, etc etc).